balancer/Dockerfile

```
@@ -0,0 +1,2 @@
+FROM nginx:alpine
```

```suggestion
FROM nginx:1.19.0-alpine
```
Produkcyjne obrazy warto spiać..